Sources/Public/Animation/LottieView.swift

```diff
@@ -438,6 +436,20 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a copy of this view with the `DotLottieConfigurationComponents`
+  /// updated to the given value.
+  ///  - Defaults to `[.imageProvider]`
+  ///  - If a component is specified here, that value in the `DotLottieConfiguration`
+  ///    of an active dotLottie animation will override any value provided via other methods.
+  public func dotLottieConfigurationComponents(
```

Comment:
Since the `DotLottieConfiguration` _overrides_ any value you pass in via `.imageProvider(...)`, `.loopMode(...)`, or `.speed(...)`, it seems like a good idea to make this behavior configurable. Otherwise it would be impossible to use a custom loop mode or speed when playing some dotLottie animations.